operate/qa/integration-tests/src/test/java/io/camunda/operate/opensearch/client/AbstractOpenSearchOperationIT.java

```diff
@@ -7,44 +7,49 @@
  */
 package io.camunda.operate.opensearch.client;
 
+import io.camunda.operate.conditions.DatabaseCondition;
 import io.camunda.operate.property.OperateProperties;
 import io.camunda.operate.schema.SchemaManager;
+import io.camunda.operate.schema.util.camunda.exporter.SchemaWithExporter;
 import io.camunda.operate.store.opensearch.client.sync.RichOpenSearchClient;
 import io.camunda.operate.util.OpensearchOperateAbstractIT;
 import io.camunda.operate.util.TestUtil;
 import java.util.function.Function;
 import org.junit.After;
 import org.junit.Before;
+import org.junit.BeforeClass;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.scheduling.concurrent.ThreadPoolTaskScheduler;
+import org.springframework.test.context.TestPropertySource;
 
+@TestPropertySource(properties = DatabaseCondition.DATABASE_PROPERTY + "=opensearch")
 public abstract class AbstractOpenSearchOperationIT extends OpensearchOperateAbstractIT {
+  private static SchemaWithExporter schemaExporterHelper;
   @Autowired protected RichOpenSearchClient richOpenSearchClient;
-
   @Autowired protected SchemaManager schemaManager;
-
   @Autowired protected OperateProperties operateProperties;
-
   @Autowired protected OpensearchTestDataHelper opensearchTestDataHelper;
-
   protected String indexPrefix;
 
+  @BeforeClass
+  public static void beforeClass() {
+    schemaExporterHelper = new SchemaWithExporter("", false);
```

Comment:
Before the test indices created by the schema manager would get the prefix that resides in operateProperties, since operateProperties prefix is not set the no prefix with exporter will match indices.

But thinking about why they used prefixes it was probably to help with concurrent tests so I will add the prefix back in